charters/DataStore/charter.md

```diff
@@ -0,0 +1,72 @@
+
+# Data Store
+
+
+## Description
+The Data Store is a scientific data sharing/publishing/distribution framework, providing file/bundle management on multiple clouds at petabyte-scale. It defines public APIs for storage, retrieval, and subscription to events that functions transparently across multiple cloud systems such as AWS and GCP.
+
+## Objective
+The objective of the Data Store group is to deliver a versioned immutable object based data repository that is highly available and scalable. Data will be replicated to at least two commercial clouds (Amazon and Google). Data will be accessible through a variety of programatic interfaces as well as a command line interface.
```

Comment:
Suggestions for the objective...highlight a top-level objective:  enable "bring compute to data": making the HCA data accessible to diverse workloads running in a public clouds, across multiple geographies, regions and vendors.    (the "replicated" and "interfaces" sentences are correct, but are really an expansion of this top-level goal)